### ISSUE TITLE
Minor display issues in IE 11

### ISSUE THREAD

**Issue by dashkevych** [dashkevych/goedemorgen#1]
User has reported that theme has a few issues in IE 11: https://wordpress.org/support/topic/minor-display-issues-in-ie-11/

**Comment by nyckelplaya** [comment]
Hello again,
Just FYI - I think those screenshots will be pulled off the server after 90 days, so please save them if you need to keep them for your records.

The navigation hover jump issue is persistent on my native install of Windows 7 with IE11. But I can reproduce it using Browserstack all the way up to Windows 10. For some reason, when using Browserstack, it only does it every-other refresh of the page, though. Windows 10 doesn't reliably show the jumping, but it does have the arrow alignment problem. 

If it would help, I'd be happy to test out updates before you roll them out on .org! Just let me know.
Thanks again--

**Comment by dashkevych** [comment]
Hi,

It would be great if you could test out the update! I'll post a fix this weekend so you can test it in your child theme.

Thanks!

**Comment by dashkevych** [comment]
Could please add this custom CSS to see if it resolves the issues in iE?
https://gist.github.com/dashkevych/b34e7bb722e159fcf6673a507ac34445

**Comment by nyckelplaya** [comment]
Whoops, I thought I replied here, but I commented on the gist link instead. Here's the feedback just in case:

Hi! Thanks for letting me test this first. I didn't do full-scale testing, but the arrow positioning is fixed where it was a problem before. Same with the text cutting off in the panel paragraph.

I was still seeing the link hover jumping (per the video), but I think I've isolated it to line 96-97 in the main parent css. If I change:
```
a:active, 
a:hover { 
  outline: 0; 
}
```

to just
```
a { 
  outline: 0; 
}
```

Then the problem is fixed. I'm not sure if that's going to conflict with your other styles, so if you'd rather let that one be I can just override it in my child css.

Will you be rolling out these changes to the theme on wp.org, or should I keep these styles in my child theme for now?

**Comment by dashkevych** [comment]
Thank you for providing a solution to fix hover jumping issue! 

Currently, you can keep these styles in your child theme. I am planning to fix these issues in the next theme version. So you can remove these styles when a new theme version will be ready.

**Comment by nyckelplaya** [comment]
Great, thank you!

**Comment by dashkevych** [comment]
The issue should be fixed in version 1.0.3.
